ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/overlay/ebgp/router-bgp.j2

```diff
@@ -47,14 +47,14 @@ router_bgp:
     {{ overlay_data.evpn_route_servers[evpn_route_server].ip_address }}:
       peer_group: {{ bgp_peer_groups.EVPN_OVERLAY_PEERS.name | arista.avd.default("EVPN-OVERLAY-PEERS") }}
       description: {{ evpn_route_server }}
-      remote_as: {{ overlay_data.evpn_route_servers[evpn_route_server].bgp_as }}
+      remote_as: "{{ overlay_data.evpn_route_servers[evpn_route_server].bgp_as }}"
 {%     if evpn_prevent_readvertise_to_server is arista.avd.defined(true) %}
-      route_map_out: RM-EVPN-FILTER-AS{{ overlay_data.evpn_route_servers[evpn_route_server].bgp_as }}
+      route_map_out: "RM-EVPN-FILTER-AS{{ overlay_data.evpn_route_servers[evpn_route_server].bgp_as }}"
```

Comment:
Since this is concatenated with a string, the quotes are not strictly needed here. They won't hurt though.

Reply:
Removed the quotes.